types/serverless/classes/Plugin.d.ts

```diff
@@ -10,34 +10,20 @@ declare namespace Plugin {
             usage?: string | undefined;
             lifecycleEvents?: string[] | undefined;
             /** @see https://www.serverless.com/framework/docs/guides/plugins/custom-commands */
-            commands?:
```

Comment:
it doesn't sound like we need to remove this

Reply:
I removed because I see no reference to it in the Serverless docs or code (version 3), maybe this was a version <= 2 thing? But I will put it back in any case.